napari/_vispy/utils/visual.py

```diff
@@ -102,7 +102,7 @@ def create_vispy_layer(layer: Layer) -> VispyBaseLayer:
     )
 
 
-def create_vispy_overlay(overlay: Overlay, **kwargs) -> List[VispyBaseOverlay]:
+def create_vispy_overlay(overlay: Overlay, **kwargs) -> VispyBaseOverlay:
```

Comment:
As you fix type annotation, please check the usage of this function and fix it or fix also docstring (docstring informs that it is for each, and code is for first).

Reply:
Fixed with [aa984ab](https://github.com/napari/napari/pull/5432/commits/aa984ab4498a7b33223f3e2567d1a9bb52b4d4f7)